Assets/Mirror/Examples/MultipleAdditiveScenes/Scripts/PlayerScore.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+namespace Mirror.Examples.MultipleAdditiveScenes
+{
+    public class PlayerScore : NetworkBehaviour
+    {
+        [SyncVar]
+        public int playerNumber;
+
+        [SyncVar]
+        public int scoreIndex;
+
+        [SyncVar]
+        public int matchIndex;
+
+        [SyncVar]
+        public uint score;
+
+        public int clientMatchIndex = -1;
+
+        void OnGUI()
+        {
+            if (!isLocalPlayer && clientMatchIndex < 0)
+                clientMatchIndex = NetworkClient.connection.identity.GetComponent<PlayerScore>().matchIndex;
```

Comment:
This should be cached, OnGui is called atleast once per frame
`NetworkClient.connection.identity.GetComponent<PlayerScore>()`

Reply:
clientMatchIndex is cached...that if is only true once.

Reply:
oh, then this check should be somewhere else (in Start()?) as that was unclear

Reply:
Can't reliably get a SyncVar value of another object in Start or OnStartClient, which is why it's here.

Reply:
maybe just put it in a function then, like `getClientMatchIndexValue()` to make OnGUI easier to read